pages/index.tsx

```diff
@@ -38,13 +38,13 @@ const Home: NextPage<{ posts: PostDocumentWithoutContent[] }> = ({ posts }) => {
     <>
       <Head>
         <title>{config.site_title}</title>
-        <meta name="description" content={config.site_description} />
         <meta property="og:title" content={config.site_keywords[1]} />
         <meta property="og:description" content={config.site_description} />
         <meta property="og:url" content={config.host_url} />
         <meta property="og:type" content="website" />
         <meta property="og:site_name" content={config.site_title} />
         <meta property="og:image" content={config.defaultImage} />
+        <meta name="description" content={config.site_description} />
```

Comment:
@ntorbinskiy this branch, is not about SEO, does it belong in here?

Reply:
No. When I was doing SEO optimization, I've got a problem with 2 titles of posts, So, I checked it in this branch and then removed. Now I understand,  that I needed to create another branch for it

Reply:
Should I turn this block back?

Reply:
Yeah, probbably remove this code in this branch, as it is not relevant

Reply:
👌